recipes/hazelcast-cpp-client/4.0.0/conandata.yml

```diff
@@ -0,0 +1,8 @@
+sources:
+  "4.0.0":
+    url: "https://github.com/hazelcast/hazelcast-cpp-client/archive/v4.0.0.zip"
+    sha256: "1bba72aec47951cefb367a99c196fd8989a2d621e527310fee9483313d347d1b"
+patches:
+  "4.0.0":
+    - patch_file: "patches/cmake_install.patch"
+      base_path: "source_subfolder"
```

Comment:
Let's use 4.0.1 and remove the patch?
I think it's great to have things already working in the upstream.

Reply:
This one is already merged 😖 https://github.com/conan-io/conan-center-index/pull/4452

Reply:
Yep, I saw that after some time and left a general comment :)
Forgot to delete this comment

Reply:
Yep, later in the review I released it was just moving the files

Reply:
I changed it as suggested so that the same recipe works for 4.0.0 and 4.0.1 releases.